src/matrFacility.ttl

```diff
@@ -147,6 +147,38 @@ somaf:Mine rdf:type owl:Class ;
            rdfs:label "mine"@en .
 
 
+###  http://sweetontology.net/matrFacility/Adit
+somaf:Mine rdf:type owl:Class ;
+           rdfs:subClassOf somaf:MiningFacility ;
+           rdfs:label "adit"@en ;
+           skos:definition "A horizontal or inclined tunnel driven from the surface for exploration, extracting ore or for dewatering a mine; also referred to as a tunnel or prospect tunnel"@en ;
+           prov:wasDerivedFrom "ISBN 0-922152-36-5" .
+
+
+###  http://sweetontology.net/matrFacility/BorrowPit
+somaf:Mine rdf:type owl:Class ;
+           rdfs:subClassOf somaf:MiningFacility ;
+           rdfs:label "borrow pit"@en ;
+           skos:definition "An area where materials including soil, clay, sand, or gravel have been excavated for use in construction at another location"@en ;
+           prov:wasDerivedFrom "ISBN 0-922152-36-5" .
+
+
+###  http://sweetontology.net/matrFacility/Quarry
+somaf:Mine rdf:type owl:Class ;
+           rdfs:subClassOf somaf:MiningFacility ;
+           rdfs:label "quarry"@en ;
+           skos:definition "An open or surface mineral excavation usually for the extraction of stone"@en ;
+           prov:wasDerivedFrom "ISBN 0-922152-36-5" .
+
+
+###  http://sweetontology.net/matrFacility/GypsumQuarry
+somaf:Mine rdf:type owl:Class ;
+           rdfs:subClassOf somaf:Quarry ;
+           rdfs:label "gypsum quarry"@en ;
+           skos:definition "An open or surface mineral working for the purpose of extracting gypsum"@en ;
+           prov:wasDerivedFrom "ISBN 0-922152-36-5" .
```

Comment:
prov:wasDerivedFrom has range prov:Entity, not obviously compatible with a string literal.   Seems to me using something like dct:source would make more sense in this context.

Reply:
Great! Also what I was looking for. I'll look across SWEET further for dct:source. I'm interested to see how we are citing sources elsewhere. That reference piece seems to be missing quite a bit right now.

Speaking of a lack of source info, I was looking at SWEET's instantiation of minerals today  in relation to how I'm working to bring Mindat minerals, along with the Strunz and Dana classifications, into our knowledge graph. Seems like an area maybe SWEET should cede territory to a domain group, focusing on perhaps clarifying higher level classes that help domains de/re-contextualize?